src/main/java/com/adobe/cq/commerce/graphql/client/impl/GraphqlClientImpl.java

```diff
@@ -127,6 +126,11 @@ public String getIdentifier() {
 
             return response;
         } else {
+            try {
+                EntityUtils.consume(httpResponse.getEntity());
```

Comment:
I'd rather use `EntityUtils.consumeQuietly()` to remove the try-catch, we don't really care here if an exception occurs because we anyway don't use the consumed entity.

Reply:
Done